packages/bls12-381/bls12-381.0.3.15/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "OCaml binding for bls12-381 from librustzcash"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381/issues"
+depends: [
+  "conf-rust" {build}
+  "ocaml" {>= "4.07"}
+  "dune" {>= "1.7"}
+  "dune-configurator" {build}
+  "ff" {>= "0.4.0" & < "0.5.0"}
+  "zarith" {>= "1.10" & < "2.0"}
+  "ctypes" {= "0.17.1"}
+  "ctypes-foreign" {= "0.4.0"}
+  "tezos-rust-libs" {= "1.0"}
```

Comment:
is there any chance of not using equality constraints here? This is very problematic in opam-repository as it forbids any patches to be used. Could this instead be something like
```
ctypes >= 0.17.1 & < 0.18.0
ctypes-foreign (this one is a meta packages so no need for a constraint)
tezos-rust-libs >= 1.0 & < 1.1 or something
```

Reply:
I relaxed the constraint on `ctypes` as suggested and remove the version constraint on `ctypes-foreign`.
For `tezos-rust-libs`, I prefer to be strict for the moment. I don't know yet the which versioning system that will be used.